run.py

```diff
@@ -12,9 +12,17 @@
 import wx
 import wx.lib.wordwrap
 import pywintypes
+import win32com.shell.shell as shell
 from wx.lib.itemspicker import ItemsPicker, IP_SORT_CHOICES, IP_SORT_SELECTED, IP_REMOVE_FROM_CHOICES
 
 vernumber = "v2.5.3"  # Version number
+ASADMIN = 'asadmin'
+
+if sys.argv[-1] != ASADMIN and not bool(ctypes.windll.advpack.IsNTAdmin(0, None)):
+    script = os.path.abspath(sys.argv[0])
+    params = ' '.join([script] + sys.argv[1:] + [ASADMIN])
+    shell.ShellExecuteEx(lpVerb='runas', lpFile=sys.executable, lpParameters=params, nShow=1)
+    sys.exit(0)
```

Comment:
OK, but for the paranoid who don't want to run a random .exe off the Internet, and/or those who wish to make code changes or enhancements or fix bugs, we'll be running it from the Python script, and that does not request admin privs as currently designed.

Putting that aside though, the API call `IsNTAdmin()` is technically much more accurate at determining whether the current process actually has administrative *rights* than `IsUserAnAdmin()`, anyway, so even if you don't like the process re-executing itself, you might want to consider changing [this](https://github.com/10se1ucgo/DisableWinTracking/blob/master/run.py#L81) line to use `IsNTAdmin()` instead, and simply refuse to run if it returns false. 

Possible situations where `IsNTAdmin()` might return false:

 - User is in the Administrator group, but click "No" on UAC prompt.
 - User is not in the Administrator group.
 - User runs Python script directly, in a command prompt that is not elevated.
 - User has some admin privileges, but not full admin rights (sometimes you'll see this on certain group policy managed configurations).

Cases where `IsUserAnAdmin()` will correctly detect insufficient privileges:

 - User does not belong to the Administrator group.

Clearly, `IsNTAdmin()` does more work here.

Note that you will also see people getting *past* `IsUserAnAdmin()` if they are a member of the Administrator group and click "No" to the UAC prompt. If they, then, try to use your program, it will fail in spectacular and unpredictable ways, which may lead to issue reports, either from people clicking "No" to the prompt, or running it from a non-elevated command prompt in a Python interpreter. Better to simply refuse to run and display a sane error message than to break in arbitrary ways and only "half-way" implement the solutions your payload has to offer (the pieces that just-so-happen to work without full elevation will work; the rest won't).